user/getting-started.md

```diff
@@ -89,6 +89,37 @@ $ echo "import foo" > sample.py
 $ bblfsh client sample.py
 ```
 
+### Overriding driver images
+
+Perhaps you need that Babelfish Server runs different driver images than the default ones. You can configure what images it should use throw the environment variable `BBLFSH_DRIVER_IMAGES`. The command `bblfsh server ` looks for this variable which must look like:
+
+    BBLFSH_DRIVER_IMAGES="language=docker-daemon:image:tag;language2=docker-daemon:image2:tag"
```

Comment:
note that docker-daemon is not the only transport possible. Any valid docker uri would serve.